lmnet/lmnet/datasets/bdd100k.py

```diff
@@ -1,16 +1,15 @@
 # -*- coding: utf-8 -*-
-import functools
-import glob
 import json
-from os.path import basename, join, splitext
-
+import glob
+import functools
 import numpy as np
+from os.path import basename, join, splitext
```

Comment:
Why did you change import order?

Reply:
So that it visually looks good. Is there a problem with order?

Reply:
(Sorry for duplicated comment) @ananno Could you apply `isort` to sort import orders?

Since aesthetic standards vary from person to person, it should not be applied as it is. We should follow some rules, and in the case of Python, I think `isort` is a safe standard.

Reply:
@tkng Agreed. I've applied `isort` and will use it from now on.